app/models/user.rb

```diff
@@ -31,17 +31,17 @@ class User < ApplicationRecord
   }
 
   after_save :make_admin
-  
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-       
-  def admin? 
+
+  def admin?
```

Comment:
I forgot that we are using enum then you don't need this method you already have it so you can remove it